### ISSUE TITLE
Can't set the temperature on TC200 Thorlabs

### ISSUE THREAD

**Issue by njrebelo** [Galvant/InstrumentKit#331]
![image](https://user-images.githubusercontent.com/47528769/154724676-bf975e43-9048-4db0-9184-92a5f80bb52c.png)

I have written this code:

import instruments as ik

com='COM7'
baud_rate = 115200

temperature_sensor=ik.thorlabs.TC200.open_serial(com,baud_rate)
print(temperature_sensor.temperature)

temperature_sensor.temperature_set(35)

and no matter what I do the erros is always the same:
Traceback (most recent call last):
  File "C:\Users\mc2\Desktop\TC200_automation.py", line 9, in <module>
    temperature_sensor.temperature_set(35)
  File "C:\Users\mc2\AppData\Local\Programs\Python\Python36\lib\site-packages\instruments\thorlabs\tc200.py", line 198, in temperature_set
    return float(response) * pq.degC
ValueError: could not convert string to float: ' 30.0elsius'

What can I do?

**Comment by scasagrande** [comment]
`temperature_set` is a property, not a function to be called

try `temperature_sensor.temperature_set = 35`

**Comment by trappitsch** [comment]
Looks to me like your device is sending back the string `30 Celsius` when you run `print(temperature_sensor.temperature)`.
With your code initialized as above in the image, up to line 6, could you run the following lines individually and post the output (error or not):

```python
print(temperature_sensor.query("tset?"))
print(temperature_sensor.query("tact?"))
```

Also: what version of `instrumentkit` do you have installed?

**Comment by scasagrande** [comment]
Or maybe its just that `tact` returns with the short form of the units, while `tset?` returns the long form?

**Comment by scasagrande** [comment]
Also this is likely 0.6 based on `pq.degC` instead of `u.degC`

**Comment by trappitsch** [comment]
@scasagrande Good point, I missed that return in the output. Also about the version. So maybe @njrebelo could still run the two queries I posted and try to set the temperature as @scasagrande pointed out above. 

If it really sends back mixed messages (`" C"` for some and `" Celsius"`), it wouldn't be too hard a fix to implement.

**Comment by scasagrande** [comment]
@trappitsch agreed

if I get a moment this weekend I'll take a look in the manual and see what it says

**Comment by scasagrande** [comment]
So in typical Thorlabs fashion the manual is very unhelpful. It doesn't give example return values for the queries.

**Comment by trappitsch** [comment]
That sounds about right... Hopefully @njrebelo can send some typical output for `tset and `tact`

**Comment by njrebelo** [comment]
Thank you so much for all the help, you guys were super helpful.
I its running now as I want it to.

![image](https://user-images.githubusercontent.com/47528769/154954452-06b0ed31-4318-466e-b706-0a52177cde2c.png)

**Comment by scasagrande** [comment]
@njrebelo could you still try this for us? `print(temperature_sensor.query("tset?"))`

**Comment by njrebelo** [comment]
![image](https://user-images.githubusercontent.com/47528769/155987155-b9fcc08b-f6b8-4b5d-bc05-23a013734003.png)
Sorry wasn´t in the lab much this week

**Comment by scasagrande** [comment]
np @njrebelo , thank you for getting us that data!

**Comment by trappitsch** [comment]
Hmm, this is interesting and weird. It really looks like `tact?` sends back ` C` for the unit and `tset?` ` Celsius`. I guess we can add a replace routine that acts on Celsius before acting on C, just to be on the safe side in case this is a settings thing...

**Comment by scasagrande** [comment]
Fixed in #333
